docs/workload/elasticsearch/elasticsearch.md

```diff
@@ -0,0 +1,573 @@
+5. [Installing and Accessing Elasticsearch](/docs/workload/elasticsearch/elasticsearch.md#installing-and-accessing-elasticsearch)
+6. [Installing Kibana](/docs/workload/elasticsearch/elasticsearch.md#installing-kibana)
+7. [Installing Fluentd-ES](/docs/workload/elasticsearch/elasticsearch.md#installing-fluentd-es)
+
+
+
+<br>
+
+
+### Install OpenEBS
+
+   If OpenEBS is not installed in your K8s cluster, this can done from [here](https://docs.openebs.io/docs/next/installation.html). If OpenEBS is already installed, go to the next step. 
+
+### Select OpenEBS storage engine
+
+A storage engine is the data plane component of the IO path of a Persistent Volume. In CAS architecture, users can choose different data planes for different application workloads based on a configuration policy. OpenEBS provides different types of storage engines and you should choose the right engine that suits your type of application requirements and storage available on your Kubernetes nodes. More information can be read from [here](https://docs.openebs.io/docs/next/overview.html#types-of-openebs-storage-engines).
+
+After OpenEBS installation, choose the OpenEBS storage engine as per your requirement. 
+- Choose **cStor**, If you are looking for replicated storage feature and other enterprise graded features such as volume expansion, backup and restore, etc. cStor configuration can be found [here](https://github.com/openebs/cstor-operators/blob/master/docs/quick.md). In this document, we will be doing Elasticsearch installation using OpenEBS cStor.
+- Choose **OpenEBS Local PV**, if you are not requiring replicated storage but high performance storage engine. The steps for deploying Prometheus operator using OpenEBS Local PV can be found [here](https://docs.openebs.io/docs/next/elasticsearch.html).
+
+
+### Configure OpenEBS cStor StorageClass
+
+The Storage Class `cstorsc-gqzxe`, which was created during cStor configuration, has been chosen to deploy Elasticsearch in the Kubernetes cluster. Please use the cStor storage class name which you had created during cStor configuration.
+
+
+###  Installing KUDO Operator
+
+In this section, we will install the KUDO operator. We will later deploy the latest available version of Elasticsearch using KUDO. 
+
+Use the latest stable version of KUDO CLI. The latest version of KUDO can be checked from [here](https://github.com/kudobuilder/kudo/releases).
+
+#### Verify if Cert-manager is installed
+
+For installing KUDO operator, the Cert-manager must be already installed in your cluster. If not, install the Cert-manager. The instruction can be found from [here](https://cert-manager.io/docs/installation/kubernetes/#installing-with-regular-manifests). Since our K8s version is v1.18.12, we have installed Cert-manager using the following command.
+
+```bash
+kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.2.0/cert-manager.yaml
```

Comment:
FYI, the latest release of Cert manager is available.

Reply:
Will use the latest version from the next deployment